src/core/main.lua

```diff
@@ -28,8 +29,8 @@ ffi.cdef[[
       extern char** argv;
 ]]
 
-_G.developer_debug = false
-debug_on_error = false
+_G.developer_debug = lib.getenv("SNABB_DEBUG") and true
+debug_on_error = _G.developer_debug
```

Comment:
@lukego Ok where do I start: the breaking change is that this sets `_G.developer_debug` to `nil` instead of `false`. Then in `core.timer`, `debug` is set to `_G.developer_debug`. The next thing I can't explain:

- When `_G.developer_debug` is `true` or `false` the value of `timer.debug` will be `true` or `false` respectively
- However when `_G.developer_debug` is `nil` the value of `timer.debug` is the following table:

```
Snabb> for k,v in pairs(timer.debug) do print(k,v) end
traceback	function: builtin#141
setlocal	function: builtin#133
getupvalue	function: builtin#134
setupvalue	function: builtin#135
upvalueid	function: builtin#136
getlocal	function: builtin#132
getregistry	function: builtin#126
getinfo	function: builtin#131
sethook	function: builtin#138
setmetatable	function: builtin#128
upvaluejoin	function: builtin#137
gethook	function: builtin#139
debug	function: builtin#140
getmetatable	function: builtin#127
setfenv	function: builtin#130
getfenv	function: builtin#129
```